net-im/discord-bin/metadata.xml

```diff
@@ -9,4 +9,8 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
+	<use>
+		<flag name="swiftshader">Install SwiftShader OpenGL libaries.</flag>
+		<flag name="system-ffmpeg">Use the system-wide <pkg>media-video/ffmpeg</pkg> instead of bundled.</flag>
```

Comment:
We don't put `.` at the end of this descriptive text.
```suggestion
		<flag name="swiftshader">Install SwiftShader OpenGL libaries</flag>
		<flag name="system-ffmpeg">Use the system-wide <pkg>media-video/ffmpeg</pkg> instead of bundled</flag>
```

Reply:
Good catch. Thanks for the suggestion/correction @juippis!